cluster/terraform_kubernetes/alertmanager.tf

```diff
@@ -0,0 +1,165 @@
+data "azurerm_key_vault" "key_vault_slack" {
+  name                = var.cluster_kv
+  resource_group_name = var.resource_group_name
+}
+data "azurerm_key_vault_secret" "slack_secret" {
+  key_vault_id = data.azurerm_key_vault.key_vault_slack.id
+  name         = "SLACK-SECRET"
+}
+
+locals {
+  alertmanager_config_path = "${path.module}/config/prometheus/alertmanager-config.yaml"
+  alertmanager_config_content = templatefile(local.alertmanager_config_path, {
+    slack_secret  = data.azurerm_key_vault_secret.slack_secret.value,
+    slack_channel = var.slack_channel
+  })
+
+  template_files = {
+    "default.tmpl" = "${path.module}/config/prometheus/alertmanager-default.yaml",
+    "slack.tmpl"   = "${path.module}/config/prometheus/alertmanager-slack.yaml"
+  }
+
+  alertmanager_templates = { for k, v in local.template_files : k => file(v) }
+}
+
+resource "kubernetes_manifest" "configmap_monitoring_alertmanager_config" {
```

Comment:
please use appropriate resource and not kubernetes_manifest. Same for all places that you are using this.

Reply:
Fixed